racingcar-java/src/main/java/step3/view/OutputView.java

```diff
@@ -0,0 +1,44 @@
+package step3.view;
+
+import step3.domain.Car;
+import step3.domain.Cars;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.*;
+
+public class OutputView {
+
+    private static final String OUTPUT_HOW_MANY_CAR = "자동차의 이름은 무엇 인가요?";
+    private static final String OUTPUT_HOW_MANY_RACING_ATTEMPT = "시도할 회수는 몇 회 인가요?";
+    private static final String OUTPUT_MOVE_CAR_POSITION = "-";
+    private static final String OUTPUT_JOINING_TEXT = ", ";
+    private static final String OUTPUT_WINNER_TEXT = "가 최종 우승했습니다.";
+
+    public static void outputHowManyCar() {
+        System.out.println(OUTPUT_HOW_MANY_CAR);
+    }
+
+    public static void outputHowManyRacingAttempt() {
+        System.out.println(OUTPUT_HOW_MANY_RACING_ATTEMPT);
+    }
+
+    public static void outputMoveCarPosition(final Cars cars) {
+        System.out.println("");
```

Comment:
```suggestion
        System.out.println();
```
사소하지만 굳이 출력하지 않아도 될 것 같네요 :)

Reply:
오오.. 감사합니다 :)